OmniBLE/PumpManager/OmniBLEPumpManager.swift

```diff
@@ -1898,8 +1894,8 @@ extension OmniBLEPumpManager: PumpManager {
         case .podSuspendedReminder:
             return PumpManagerAlert.suspendInProgress(triggeringSlot: slot)
         case .expirationReminder:
-            let timeToExpiry = TimeInterval(hours: expiresAt.timeIntervalSince(dateGenerator()).hours.rounded())
-            return PumpManagerAlert.userPodExpiration(triggeringSlot: slot, scheduledExpirationReminderOffset: timeToExpiry)
+            let offset = state.scheduledExpirationReminderOffset ?? 0
+            return PumpManagerAlert.userPodExpiration(triggeringSlot: slot, scheduledExpirationReminderOffset: offset)
```

Comment:
I don't think this is right. Assuming that the pod is in range and we get the alert immediately, this might be ok. But if the pod is out of range for an extended period, and when it comes back in range and we see the alert, then the offset until expiration might be significantly different.

Reply:
Thanks for the review! I'd totally misread this. Now, a `.userPodExpiration` will only be returned when there is a scheduled offset, and otherwise, the `timeToExpiry` will be passed for display in the `contentBody` of the alert.